frame/aura/src/lib.rs

```diff
@@ -179,8 +187,12 @@ impl<T: Config> OneSessionHandler<T::AccountId> for Pallet<T> {
 		if changed {
 			let next_authorities = validators.map(|(_, k)| k).collect::<Vec<_>>();
 			let last_authorities = Self::authorities();
-			if next_authorities != last_authorities {
-				Self::change_authorities(next_authorities);
+			if last_authorities != next_authorities {
+				let bounded = <WeakBoundedVec<_, T::MaxAuthorities>>::force_from(
+					next_authorities,
+					Some("AuRa new session"),
+				);
```

Comment:
We just assume that the previous authorities fit in the bounded vec?
Should we at least add some logging so we notice if this assumption fails?

Reply:
What I was intending to do is to let the authorities still be included when it exceeds the maximum. One would simply not be able to add any more additional authorities, until the number of authorities gets reduced below the maximum.

Also, `force_from` does provide logging if the length exceeds the maximum.